api/templates/military-history.xml

```diff
@@ -7,9 +7,9 @@
     <Branch>{{radio $Item.Service}}</Branch>
     <Comment></Comment>
     {{monthYearDaterange $Item.Dates}}
-    <!-- <GuardLocation> -->
-    <!--   <State>DQ</State> -->
-    <!-- </GuardLocation> -->
+    {{if or (radio $Item.Service | eq "ArmyNationalGuard") (radio $Item.Service | eq "ArmyNationalGuard")}}
+    <GuardLocation>{{location $Item.GuardLocation}}</GuardLocation>
```

Comment:
Yeah, we need to do something different here per the XSD:
```
<GuardLocation><State>{{$Item.GuardLocation.props.state}}</State></GuardLocation>
```

Reply:
Also there's a copy and paste error – Army National Guard listed twice.

Reply:
Thanks, xml updated